src/components/Download.jsx

```diff
@@ -0,0 +1,133 @@
+import AppStore from '/AppStore.svg'
+import GooglePlay from '/GooglePlay.svg'
+import '/weightLifting.svg'
+import '/weightTablet.svg'
+import '/weightDesk.svg'
+import styled from 'styled-components'
+
+export const Download = () => {
+  return (
+    <StyledDownload>
+      <picture className="weightpic">
```

Comment:
We are wondering why you chose picture instead of img element.

// Sofie & Wen